tests/constants.py

```diff
@@ -17,7 +17,8 @@
 MAX_COMBINED_LENGTH_TOKENS: int = 70_000
 
 CHARACTERS_PER_TOKEN: int = 4
-WORD_THAT_EQUALS_ONE_TOKEN = "hello "  # noqa: S105
+# 4 character (3 character + 1 space) = 1 token
+WORD_THAT_EQUALS_ONE_TOKEN = "orb "  # noqa: S105
```

Comment:
`test_prompt_within_context_limit_returns_response` was failing as "hello " was 1 token but 6 characters. As our backend token count logic uses 2 approaches - tokenizer and total_characters // 4, `"hello " * get_default_context_limit` translated to > ~70K tokens, which led to the test failing.

For safer side, until we unify our token counting logic, using "orb " (4 characters, 1 token) for `WORD_THAT_EQUALS_ONE_TOKEN`